tests/pcieutil_test.py

```diff
@@ -199,6 +202,16 @@ def test_aer_option_device(self):
         result = runner.invoke(pcieutil.cli.commands["pcie-aer"].commands["correctable"], ["-d", "0:1.0"])
         assert result.output == pcieutil_pcie_aer_correctable_dev_output
 
+    def test_load_pcie_module_warning(self): 
+        stdout = sys.stdout
+        sys.stdout = result = StringIO()
+        try:
+            pcieutil.load_platform_pcieutil()
+        except ImportError:
+            pass
+        sys.stdout = stdout
+        assert pcieutil_load_module_warning_msg not in result.getvalue()
```

Comment:
@cytsao1 what does this ASSERT confirm? There could be platform which may not see this import Error and in which case this ASSERT may fail?

Reply:
@cytsao1 `result` value is empty string at the start, so when the test run at line 209, `result` is not updated at all and at line 213 ofcourse `pcieutil_load_module_warning_msg` won't match `result` value, so i don't understand why even run `pcieutil.load_platform_pcieutils()` at line 209?

Reply:
sys.stdout is updated to be same IO stream as result, so running load_platform_pcieutil output will be captured in result instead of output to console stdout.
After, the stdout is set back to the sys.stdout stream.

Reply:
Example of how it would look without this PR
```
>>> import sys
>>> from io import StringIO
>>> import pcieutil.main as pcieutil
>>> stdout = sys.stdout
>>> sys.stdout = result = StringIO()
>>> pcieutil.load_platform_pcieutil()
>>> sys.stdout = stdout
>>> result.getvalue()
"Failed to load platform Pcie module. Error : No module named 'sonic_platform.pcie', fallback to load Pcie common utility.\n"
```

Reply:
@cytsao1 thanks for clarifying....did not realize that stdout was captured in a string output...